Missing provider error

When running `npm run test` in the hello world I get a missing provider error. Looks like web3 is missing @Shirikatsu @maxrobot @daragao @saronimo 

```
vagrant@ubuntu-xenial:~$ npm run test
npm ERR! path /home/vagrant/package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/home/vagrant/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2019-01-29T16_23_00_946Z-debug.log
vagrant@ubuntu-xenial:~$ cd ion/
vagrant@ubuntu-xenial:~/ion$ npm run test

> ion-truffle@1.0.0 test /home/vagrant/ion
> truffle test

Compiling ./contracts/EventVerifier.sol...
Compiling ./contracts/Ion.sol...
Compiling ./contracts/IonCompatible.sol...
Compiling ./contracts/Migrations.sol...
Compiling ./contracts/functional/Function.sol...
Compiling ./contracts/functional/Trigger.sol...
Compiling ./contracts/functional/TriggerEventVerifier.sol...
Compiling ./contracts/libraries/ECVerify.sol...
Compiling ./contracts/libraries/PatriciaTrie.sol...
Compiling ./contracts/libraries/RLP.sol...
Compiling ./contracts/libraries/SafeMath.sol...
Compiling ./contracts/libraries/SolidityUtils.sol...
Compiling ./contracts/mock/MockIon.sol...
Compiling ./contracts/mock/MockStorage.sol...
Compiling ./contracts/mock/MockValidation.sol...
Compiling ./contracts/storage/BlockStore.sol...
Compiling ./contracts/storage/EthereumStore.sol...
Compiling ./contracts/validation/Clique.sol...
Compiling ./test/PatriciaTrieTest.sol...
Compiling truffle/Assert.sol...

Compilation warnings encountered:

/home/vagrant/ion/contracts/storage/BlockStore.sol:55:5: Warning: Modifiers of functions without implementation are ignored.
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) onlyIon onlyRegisteredChains(_chainId);
    ^---------------------------------------------------------------------------------------------------------------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:399:17: Warning: The use of labels is deprecated. Please use "if", "switch", "for" or function calls instead.
                tag_loop:
                ^------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:399:17: Warning: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
                tag_loop:
                ^------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:400:17: Warning: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
                jumpi(end, eq(i, words))
                ^----------------------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:406:17: Warning: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
                jump(tag_loop)
                ^------------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:407:17: Warning: The use of labels is deprecated. Please use "if", "switch", "for" or function calls instead.
                end:
                ^-^
,/home/vagrant/ion/contracts/libraries/RLP.sol:407:17: Warning: Jump instructions and labels are low-level EVM features that can lead to incorrect stack access. Because of that they are discouraged. Please consider using "switch", "if" or "for" statements instead.
                end:
                ^-^
,/home/vagrant/ion/contracts/storage/BlockStore.sol:55:5: Warning: No visibility specified. Defaulting to "public".
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) onlyIon onlyRegisteredChains(_chainId);
    ^---------------------------------------------------------------------------------------------------------------^
,/home/vagrant/ion/contracts/storage/EthereumStore.sol:53:5: Warning: No visibility specified. Defaulting to "public".
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob)
    ^ (Relevant source part starts here and spanvagrant@ubuntu-xenial:~$ npm run test
npm ERR! path /home/vagrant/package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/home/vagrant/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /`s across multiple lines).
,/home/vagrant/ion/contracts/storage/EthereumStore.sol:178:5: Warning: No visibility specified. Defaulting to "public".
    function verifyProof(bytes _value, bytes _parentNodes, bytes _path, bytes32 _hash) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/functional/TriggerEventVerifier.sol:29:9: Warning: Unused local variable.
        bytes memory contractEmittedEvent = RLP.toData(log[0]);
        ^-------------------------------^
,/home/vagrant/ion/contracts/functional/TriggerEventVerifier.sol:30:9: Warning: Unused local variable.
        RLP.RLPItem[] memory topics = RLP.toList(log[1]);
        ^-------------------------^
,/home/vagrant/ion/contracts/mock/MockIon.sol:8:5: Warning: No visibility specified. Defaulting to "public".
    constructor(bytes32 _id) Ion(_id) {}
    ^----------------------------------^
,/home/vagrant/ion/contracts/mock/MockIon.sol:18:5: Warning: No visibility specified. Defaulting to "public".
    function addChain(address _storageAddress, bytes32 _chainId) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/mock/MockStorage.sol:22:5: Warning: No visibility specified. Defaulting to "public".
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/mock/MockStorage.sol:22:23: Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) {
                      ^--------------^
,/home/vagrant/ion/contracts/mock/MockStorage.sol:22:61: Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
    function addBlock(bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) {
                                                            ^--------------^
,/home/vagrant/ion/contracts/mock/MockValidation.sol:13:5: Warning: No visibility specified. Defaulting to "public".
    function SubmitBlock(address _storageAddress, bytes32 _chainId, bytes32 _blockHash, bytes _blockBlob) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/validation/Clique.sol:155:98: Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
    function addGenesisBlock(bytes32 _chainId, address[] _validators, bytes32 _genesisBlockHash, address _storeAddr) internal {
                                                                                                 ^----------------^
,/home/vagrant/ion/contracts/validation/Clique.sol:326:5: Warning: No visibility specified. Defaulting to "public".
    function getValidators(bytes32 _chainId, bytes32 _blockHash) constant returns (address[]) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/validation/Clique.sol:330:5: Warning: No visibility specified. Defaulting to "public".
    function getProposal(bytes32 _chainId, bytes32 _blockHash, address _candidate) constant returns (uint256) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/test/PatriciaTrieTest.sol:9:5: Warning: No visibility specified. Defaulting to "public".
    function testSucceedVerifyNestedNodes() returns (bool) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/test/PatriciaTrieTest.sol:43:5: Warning: No visibility specified. Defaulting to "public".
    function testFailVerifyNestedNodes() returns (bool) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/Ion.sol:42:5: Warning: Function state mutability can be restricted to view
    function isContract(address _addr) internal returns (bool) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/libraries/PatriciaTrie.sol:114:5: Warning: Function state mutability can be restricted to pure
    function checkNodeValue(bytes _expected, bytes _nodeValue) private returns (bool) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/libraries/RLP.sol:171:5: Warning: Function state mutability can be restricted to view
    function toBytes(RLPItem memory self) internal returns (bytes memory bts) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/libraries/RLP.sol:183:5: Warning: Function state mutability can be restricted to view
    function toData(RLPItem memory self) internal returns (bytes memory bts) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/libraries/RLP.sol:214:5: Warning: Function state mutability can be restricted to view
    function toAscii(RLPItem memory self) internal returns (string memory str) {
    ^ (Relevant source part starts here and spans across multiple lines).
,/home/vagrant/ion/contracts/libraries/RLP.sol:400:17: Warning: Function declared as view, but this expression (potentially) modifies the state and thus requires non-payable (the default) or payable.
                jumpi(end, eq(i, words))
                ^----------------------^
,/home/vagrant/ion/contracts/libraries/RLP.sol:406:17: Warning: Function declared as view, but this expression (potentially) modifies the state and thus requires non-payable (the default) or payable.
                jump(tag_loop)
                ^------------^
,/home/vagrant/ion/contracts/validation/Clique.sol:185:5: Warning: Function state mutability can be restricted to view
    function checkSignature(bytes32 _chainId, bytes _extraData, bytes _rlpBlockHeader, bytes32 _parentBlockHash) internal returns (bool) {
    ^ (Relevant source part starts here and spans across multiple lines).

Error: Missing parameter: provider
    at Function.throwIfMissing (/home/vagrant/ion/node_modules/web3/node_modules/web3-core/dist/web3-core.cjs.js:148:13)
    at Web3.AbstractWeb3Module (/home/vagrant/ion/node_modules/web3/node_modules/web3-core/dist/web3-core.cjs.js:17:107)
    at new Web3 (/home/vagrant/ion/node_modules/web3/dist/web3.cjs.js:30:68)
    at Object.<anonymous> (/home/vagrant/ion/test/clique.js:26:14)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at /home/vagrant/ion/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/vagrant/ion/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/home/vagrant/ion/node_modules/mocha/lib/mocha.js:536:10)
    at /home/vagrant/ion/node_modules/truffle/build/webpack:/packages/truffle-core/lib/test.js:125:1
    at process._tickCallback (internal/process/next_tick.js:68:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ion-truffle@1.0.0 test: `truffle test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ion-truffle@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2019-01-29T16_23_24_889Z-debug.log

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 os: linux
-dist: trusty
+dist: 1:0.5.1-0ubuntu1~trusty
 
 matrix:
  include:

--- a/package.json
+++ b/package.json
@@ -32,9 +32,9 @@
     "json-bigint-string": "^1.0.0",
     "solidity-coverage": "^0.5.0",
     "truffle": "^4.1.7",
-    "web3": "^1.0.0-beta.33",
-    "web3-eth-abi": "^1.0.0-beta.33",
-    "web3-utils": "^1.0.0-beta.33"
+    "web3": "1.0.0-beta.33",
+    "web3-eth-abi": "1.0.0-beta.33",
+    "web3-utils": "1.0.0-beta.33"
   },
   "scripts": {
     "testrpc": "ganache-cli --port 8545 --gasLimit 0xFFFFFFFFFFF --gasPrice 0 --defaultBalanceEther 99999999999 --networkId 1234",

